app/currency.rb

```diff
@@ -71,7 +71,7 @@
   name: 'USDC (STE)',
   symbol: '$',
   subunit: 'cents',
-  subunit_to_unit: 1_0_000_000,
+  subunit_to_unit: 10_000_000,
```

Comment:
Fine with this but I did it intentionally the other way to somewhat show the decimal place. Maybe `1_0000000` would have been better.